examples/digital_fingerprinting/production/README.md

```diff
@@ -27,19 +27,17 @@ Key Features:
  * Can be deployed to Kubernetes using provided Helm charts
  * Uses many customized stages to maximize performance.
 
-## Build the Morpheus container
-This is necessary to get the latest changes needed for DFP. From the root of the Morpheus repo:
-```bash
-./docker/build_container_release.sh
-```
-
 ## Building and Running via `docker compose`
 ### Build
 ```bash
 cd examples/digital_fingerprinting/production
-export MORPHEUS_CONTAINER_VERSION="$(git describe --tags --abbrev=0)-runtime"
 docker compose build
 ```
+You can optionally specify the version of the morpheus-dfp conda package for the pipeline via the build-arg. The version defaults to the latest release.
+```bash
+docker compose build --build-arg MORPHEUS_DFP_VERSION=24.10.00a
```

Comment:
We should probably go back to using `git describe` to drive this, otherwise it will need to be added to `ci/release/update-version.sh`

Reply:
Makes sense. But in that case I should probably leave morpheus-dfp installation in this docker file.